### ISSUE TITLE
column information from query

### ISSUE THREAD

**Issue by Gary-Ascuy** [w1nk/node-odbc#91]
Is there a way to retrieve all column information for a statement (SQLHSTMT hStmt) in nodejs API, I think this would be helpful expose that information

**Comment by wankdanker** [comment]
It is not currently exposed but is a great idea.  Is there anything specific you are looking for? I think data type, length/size would be helpful.

**Comment by Gary-Ascuy** [comment]
yep, thanks in advance (y)

**Comment by Gary-Ascuy** [comment]
Yes, in addition to type and length/size fields, I think column names would be a great addition

**Comment by Zilev** [comment]
+1  Yeah I need type and column names also, thanks in advance :smile:

**Comment by mtrezzi** [comment]
Yes it's a great idea!
If a query doesn't return any data I cannot guess field types and names.
:+1:
